roles/openshift_version/tasks/main.yml

```diff
@@ -119,30 +127,42 @@
 
 - fail:
     msg: openshift_version role was unable to set openshift_version
```

Comment:
Actually these three are more of a "that shouldn't happen" kind of check so I'm not really sure what to recommend.